mod_updater.py

```diff
@@ -241,9 +241,9 @@ def _retrieve_mod_metadata(self, mod: str):
         """
         data = self.mods[mod]
         mod_url = self.mod_server_url + '/api/mods/' + mod + '/full'
-        with requests.get(mod_url) as req:
-            if req.status_code == 200:
-                data['metadata'] = req.json()
+        req = requests.get(mod_url)
+        if req.status_code == 200:
+            data['metadata'] = req.json()
```

Comment:
This actually changes what it's doing -- you need to wrap this in a `try: finally:` block to close the request.